app/graphql/mutations/projects/delete.rb

```diff
@@ -0,0 +1,18 @@
+module Mutations
+  module Projects
+    class Delete < Mutations::BaseMutation
+      argument :project_id, ID, required: true, loads: Types::Models::Project
+
+      field :project, Types::Models::Project, null: true
+
+      def resolve(project:)
+        authorize(project)
+        project.delete
```

Comment:
In future tasks going to be added. They will have dependency "destroy" in project model. So now we have a struggle that this dependency will be missed here because #delete method only performs direct sql query.
Maybe #destroy is more convenient here.
Catching of failed destroy must be applied if you choose to use #destroy.